storage/rocksdb/rdb_i_s.cc

```diff
@@ -845,17 +845,22 @@ static int rdb_i_s_compact_stats_fill_table(
       continue;
     }
 
-    std::map<std::string, double> props;
+    std::map<std::string, std::string> props;
```

Comment:
What additional keys are returned when switching to the string version of GetMapProperty? The information schema table is still returning a double though, so the additional keys would need to have values that are doubles, which seems odd that they're not returned by the double version of GetMapProperty in the first place.

Reply:
Now all keys are returned with some prefixes. The old values now have "compaction." key prefix and new ones have "io_stalls." prefix. This method now skips keys with prefixes other than "compaction."

I've tested INFORMATION_SCHEMA.ROCKSDB_COMPACTION_STATS table and see that it contains the same data it contained before.

Reply:
What would be returned in I_S.COMPACTION_STATS without this change? Does calling GetMapProperty() using std::map<std::string, std::double> not return all of the keys with "compaction." key prefix?

Reply:
I've updated summary to highlight what rocksdb changes were done relevant to this PR.